src/Core/Addon/Theme/ThemeExporter.php

```diff
@@ -65,9 +68,11 @@ public function __construct(
     }
 
     /**
+     * Exports a theme into a zip file
+     *
      * @param Theme $theme
      *
-     * @return false|string
+     * @return string Path to the exported zip file
```

Comment:
```suggestion
     * @return false|string Path to the exported zip file
```

https://www.php.net/realpath